### PR TITLE
Added net.standard 1.0 and PCL supporting

### PR DESCRIPTION
**Pull request by rotorgames**

### CODE REVIEW

File: PanCardView/CardsView.cs [hunk 1]
```diff
@@ -1731,7 +1732,7 @@ private void ExecutePreventInvalidOperationException(Action action, int restartC
                             ExecutePreventInvalidOperationException(action, ++restartCount);
                             return;
                         }
-                        Console.WriteLine("CardsView: Couldn't handle InvalidOperationException");
+                        Debug.WriteLine("CardsView: Couldn't handle InvalidOperationException");
```
Comment (AndreiMisiukevich) [inline PanCardView/CardsView.cs]: Can we preserve ```Console.Writeline``` and remove ```using System.Diagnostics;```

Comment (rotorgames) [reply PanCardView/CardsView.cs]: No, We can't. Console is not supported in net.standard1.0

Comment (rotorgames) [reply PanCardView/CardsView.cs]: System.Console package is available only in net.standard1.3 or higher.

Comment (rotorgames) [reply PanCardView/CardsView.cs]: I can save `Console` for net.standard 2.0 if you want. But I can't see a big reason for it.

Comment (AndreiMisiukevich) [inline PanCardView/CardsView.cs]: How about this?
```suggestion
#if NETSTANDARD2_0
                        Console.WriteLine("CardsView: Couldn't handle InvalidOperationException");
#endif
```

File: PanCardView/CardsView.cs [hunk 2]
```diff
@@ -18,6 +18,7 @@
 using PanCardView.EventArgs;
 using PanCardView.Delegates;
 using System.ComponentModel;
+using System.Diagnostics;
```
Comment (AndreiMisiukevich) [inline PanCardView/CardsView.cs]: just removing it
```suggestion
```

File: PanCardView/Resources/ResourcesInfo.cs [hunk 3]
```diff
@@ -11,7 +11,7 @@ public static class ResourcesInfo
         private const string BlackRightArrowResourceName = "rightArrowBlack.png";
         private const string BlackLeftArrowResourceName = "leftArrowBlack.png";
 
-        private static Assembly ResourceAssembly => typeof(ResourcesInfo).Assembly;
+        private static Assembly ResourceAssembly => typeof(ResourcesInfo).GetTypeInfo().Assembly;
```
Comment (AndreiMisiukevich) [inline PanCardView/Resources/ResourcesInfo.cs]:
```suggestion
        private static Assembly ResourceAssembly
            => typeof(ResourcesInfo).
#if !NETSTANDARD2_0
            GetTypeInfo().
#endif
            Assembly;
```

Comment (rotorgames) [reply PanCardView/Resources/ResourcesInfo.cs]: I think it is not necessary. GetTypeInfo works fine for each target.

**Pull request merged**